test/integration/test_serialization_bc.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD 3-Clause license found in the
+# LICENSE file in the root directory of this source tree.
+
+import unittest
+
+import torch
+from torch.testing._internal import common_utils
+from torch.testing._internal.common_utils import (
+    TestCase,
+    run_tests,
+)
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from torchao.utils import _is_fbgemm_genai_gpu_available, is_sm_at_least_90
+
+_MODEL_NAMES = [
+    "torchao-testing/opt-125m-float8dq-row-fbgemm",
```

Comment:
I think model name here should specify the relevant versions

also, IMO this should be a toy model with a single layer with matching done on the layer output, to make it 100x easier to debug when things do go wrong.  It's fine to also have a real model and match tokens, but I think it's more important to have a toy model.

Reply:
ok, but can we include the relevant versions in the name today?